lms/templates/certificates/_accomplishment-related.html

```diff
@@ -0,0 +1,49 @@
+<div class="wrapper-accomplishment-related">
```

Comment:
NOTE: future-thinking UX here of associating other course-based accomplishments with the main one featured (certificates in this case). We can remove if not needed now.